x-pack/legacy/plugins/siem/public/components/skeleton_row/index.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pure } from 'recompose';
+import styled, { css } from 'styled-components';
+
+const Row = styled.div<{ rowHeight?: string; rowPadding?: string }>`
+  ${({ rowHeight, rowPadding, theme }) => css`
+    border-bottom: ${theme.eui.euiBorderWidthThin} solid ${theme.eui.euiColorLightShade};
+    display: flex;
+    height: ${rowHeight ? rowHeight : theme.eui.euiSizeXL};
+    padding: ${rowPadding
+      ? rowPadding
+      : theme.eui.paddingSizes.s + ' ' + theme.eui.paddingSizes.xs};
+  `}
+`;
+Row.displayName = 'Row';
+
+const Cell = styled.div<{ cellColor?: string; cellMargin?: string }>`
+  ${({ cellColor, cellMargin, theme }) => css`
+    background-color: ${cellColor ? cellColor : theme.eui.euiColorLightestShade};
+    border-radius: 2px;
+    flex: 1;
+
+    & + & {
+      margin-left: ${cellMargin ? cellMargin : theme.eui.gutterTypes.gutterSmall};
+    }
+  `}
+`;
+Cell.displayName = 'Cell';
+
+export interface SkeletonRowProps {
+  cellColor?: string;
+  cellCount?: number;
+  cellMargin?: string;
+  rowHeight?: string;
+  rowPadding?: string;
+  style?: object;
+}
+
+export const SkeletonRow = pure<SkeletonRowProps>(
```

Comment:
We're all so used to using `pure`! 😄  Since this is new code, consider using `React.memo` here, and whenever a shallow comparison of props is sufficient to prevent unnecessary re-renders

Reply:
👍